### PR TITLE
Adding links to the new wasm article on Medium

### PR DESCRIPTION
**Pull request by sfshaza2**
As it says.

cc @kevmoo

### CODE REVIEW

File: src/content/platform-integration/web/wasm.md [hunk 1]
```diff
@@ -33,6 +34,14 @@ Revisit this page for the latest updates.
 [`package:web`]: {{site.pub-pkg}}/web
 [`dart:js_interop`]: {{site.dart.api}}/{{site.dart.sdk.channel}}/dart-js_interop 
 
+:::note
+For information on how to optimize web loading speed,
+check out the (free) article on Medium,
+[Best practices for optimizing Flutter web loading speed][article].
+
+[article]: {{site.flutter-medium}}/best-practices-for-optimizing-flutter-web-loading-speed-7cc0df14ce5c
+:::
```
Comment (atsansone) [inline src/content/platform-integration/web/wasm.md:43]: **question:** As this is repeated in [another file](https://github.com/flutter/website/pull/10516/files#diff-40540badc69a9b7240fba3b99d24ebfff2735f812a45aa6d419199f33beb2986R15-R21), why not make this an include / render instead?

File: src/content/platform-integration/web/wasm.md [hunk 2]
```diff
@@ -64,12 +73,12 @@ the `beta` channel (preferred) or `main`.
 
 To learn more about Flutter build release channels and how to switch to
 the `beta` channel, check out the
-[Flutter wiki](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels).
+[Flutter wiki]({{site.github}}/flutter/flutter/wiki/Flutter-build-release-channels).
```
Comment (atsansone) [inline src/content/platform-integration/web/wasm.md:76]: **question:** Here and elsewhere, you point to a Wiki article that's almost 6 years old. Some of the information is [covered in the docs now](https://docs.flutter.dev/testing/build-modes) which is also 5+ years old. Should we link to the docs page first and look at updating it and bring some of the Wiki content into docs?

File: src/content/platform-integration/web/wasm.md [hunk 3]
```diff
@@ -132,7 +142,7 @@ package root.
 
 ### Serve the output locally with an HTTP server
 
-If you don't have a local HTTP server installed, you can use the
+If you don't have a local HTTP server installed, you can use
```
Comment (atsansone) [inline src/content/platform-integration/web/wasm.md:145]: **question:** Why remove the `the`? This reads awkwardly now.

File: src/content/tools/devtools/deep-links.md
Comment (atsansone) [inline src/content/tools/devtools/deep-links.md:1]: **question:** This page appears to be pulled from another PR: https://github.com/flutter/website/pull/10509. Maybe you need to rebase against `upstream/main` and the force push clear this out.

File: src/content/tools/devtools/overview.md
Comment (atsansone) [inline src/content/tools/devtools/overview.md:1]: **question:** This page appears to be pulled from another PR: https://github.com/flutter/website/pull/10509. Maybe you need to rebase against `upstream/main` and the force push clear this out.

File: src/content/platform-integration/web/wasm.md [hunk 4]
```diff
@@ -64,12 +73,12 @@ the `beta` channel (preferred) or `main`.
...
 
-To then ensure you are running the latest version,
+To ensure you are running the latest version,
 run `flutter upgrade`.
 
-To verify if your Flutter install supports Wasm,
+To verify that your Flutter install supports Wasm,
 run `flutter build web --help`.
 
 At the bottom of the output, you should find experimental Wasm options like:
```
Comment (atsansone) [inline src/content/platform-integration/web/wasm.md:84]: **question:** Wouldn't this be conditional on having Wasm support?

```suggestion
If your Flutter install supports Wasm, the response to this command should show experimental Wasm options.
```

**Review by atsansone — COMMENTED** [review]
There appear to be some extra files in this PR and a few other small questions. PTAL.

File: src/content/platform-integration/web/wasm.md (hunk 2 above)
Comment (parlough) [reply src/content/platform-integration/web/wasm.md:76]: Yes this has always been an awkward discrepency. We should update the docs with any relevant information. I'll defer that to separate, follow-up work though as those links are across the website.

File: src/content/platform-integration/web/wasm.md (hunk 4 above)
Comment (parlough) [reply src/content/platform-integration/web/wasm.md:84]: Hmm, might be good to clarify that, but for now it seems the conditionality is implied by the previous sentence. Feel free to open a new PR with this change as I'd like to get this landed. Thanks!

**Review by parlough — APPROVED** [review]
I pushed a few fixes to the review comments and fixed the accidental included commit. Thanks!
